boudicca.events/eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/WissensturmCollector.kt

```diff
@@ -23,9 +23,14 @@ class WissensturmCollector : TwoStepEventCollector<Pair<String, Document>>("wiss
         var date = LocalDate.now(ZoneId.of("Europe/Vienna"))
 
         //only collect 6 months for now
-        for (i in 1..6) {
+        for (ignored in 1..6) {
```

Comment:
ignored?

Reply:
i cannot use "_" as a name for a variable in kotlin in a for loop, so i have to use "ignored" to signify that i don't care about it. otherwise i have an unused variable